### PR TITLE
What's New and Upgrade Steps for 6.0

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by brandonpage** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a summary in the Pull Request description</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_ci/circleci">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -23,17 +23,41 @@ This pulls submodule dependencies from github.
...
+
+**SmartSync Enhancements**
+- Beginning in Mobile SDK 6.0, you can define sync configuration files and assign names to sync configurations.
+- You can use sync names to run, edit, or delete a saved sync operation.
+- You can define “sync down” and “sync up” operations through configuration files rather than code.
+- To define sync operations for the default global store, provide a file named `globalsyncs.json`.
+- To define sync operations for the default user store, provide a file named `usersyncs.json`.
+
+**Other Enhancements**
+- Mobile SDK 6.0 adds support for a developer settings screen in DEBUG builds.
```
Comment (wmathurin) [inline README.md]: New in Mobile SDK 6.0, we are adding dev support screens. You do not need to write any code to use them. To access the dev support dialog, use the ⌘m keyboard shortcut when your app is running in the Android emulator. Alternatively, you can run the command: adb shell input keyevent 82.

File: UPGRADE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-## Upgrade steps from v. 5.2.x to v. 5.3 for native/hybrid/react native apps
+## Upgrade steps from v. 5.3.x to v. 6.0 for native/hybrid/react native apps
```
Comment (wmathurin) [inline UPGRADE.md]: We should mention that there is a new cli for generating hybrid apps (forcehybrid) and another  one for generating react native apps (forcereact).
Should we mention the sfdx plugin as well?

File: README.md (hunk 1 above)
Comment (bhariharan) [reply README.md]: Will do.

File: UPGRADE.md (hunk 2 above)
Comment (bhariharan) [reply UPGRADE.md]: Will do.

**Comment by bhariharan** [comment]
@wmathurin Done.

**Comment by bhariharan** [comment]
Got it reviewed by Richard offline.

**Pull request merged**